### DISCUSSION TITLE
how to disable test explorer run icon

### DISCUSSION THREAD

**Discussion by Young-Flash** [rust-lang/rust-analyzer#17906] [Q&A] [answered]
there are `Run test` codelen and test explorer run icon, is there a way to disable the latter?

<img width="225" alt="image" src="https://github.com/user-attachments/assets/f0214d52-8d40-4936-a361-a16a9ce08422">

cc @HKalbasi

**Comment by lnicola** [comment]
Without disabling the test explorer, right?

**Reply by Young-Flash** [reply]
yeah

**Reply by Young-Flash** [reply, marked as answer]
figured it out, add `"testing.gutterEnabled": false` in settings.json, thanks
